pkg/network/server.go

```diff
@@ -282,6 +279,48 @@ func newServerFromConstructors(config ServerConfig, chain Ledger, stSync StateSy
 	return s, nil
 }
 
+type blockAdapter struct {
```

Comment:
Move all adapters to a separate file. Declare a variable to ensure it implements proper interface, like `var _ = (queue.Blockqueuer[*block.Block])(&blockAdapter{})